src/main/java/com/kodilla/ecommercee/domain/ProductDto.java

```diff
@@ -8,6 +8,7 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class ProductDto {
-    private long productId;
     private String productName;
+    private String description;
+    private double price;
```

Comment:
cena tylko w BigDecimal, double nie nadaje się do liczenia kasy :D 
poczytaj; https://pl.wikipedia.org/wiki/IEEE_754 
https://golinski.faculty.wmi.amu.edu.pl/zemn/lecture01.pdf

Reply:
Właśnie moduł z ciekawym opisem BigDecimal przerobiłem i wiem już jakie to fajne, i że double to kradnie ;)